src/openfermion/utils/_operator_utils.py

```diff
@@ -462,10 +478,14 @@ def save_operator(operator, file_name=None, data_directory=None,
     else:
         raise TypeError('Operator of invalid type.')
 
-    tm = operator.terms
-    with open(file_path, 'wb') as f:
-        marshal.dump((operator_type, dict(zip(tm.keys(),
-                                              map(complex, tm.values())))), f)
+    if human_readable:
+        with open(file_path, 'w') as f:
+            f.write(operator_type + ": " + str(operator))
```

Comment:
I think we'll want to do `:\n` instead of `: `. Can you add a test with small operator that explicitly includes what the saved format looks like? I.e., the test should compare the saved file with an explicit string.